packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -52,6 +53,11 @@ class FlexibleSpaceBar extends StatefulWidget {
   /// Defaults to being adapted to the current [TargetPlatform].
   final bool centerTitle;
 
+  /// Adds padding to the [title] before scaling is applied.
+  ///
+  /// Defaults to being the amount to exclude the leading.
```

Comment:
Can you explain the default a little better here? I'm not sure I understand what I'll get by default.